lib/app/utils/measure_operations.rb

```diff
@@ -159,5 +159,26 @@ def get_valueset_urls(library)
         .uniq
         .to_a
     end
+
+    def get_data_requirements_queries(data_requirement)
+      # hashes with { endpoint => FHIR Type, params => { queries } }
+      ret_val = []
+      dr_with_filters = data_requirement.select { |dr| !dr.codeFilter.nil? && !dr.codeFilter.first.nil? }
+
+      dr_with_filters.each do |dr|
```

Comment:
If you wanted to be really ruby-ish about it, you could remove `ret_val` entirely, change this to `.map` instead of `.each`, and change that last `ret_val.push` line to just `q`

Reply:
And delete the `ret_val` line at the end of the method

Reply:
did ya one better

Reply:
_Rubying intensifies_